### DISCUSSION TITLE
Is it okay to push .vscode/settings.json to github?

### DISCUSSION THREAD

**Discussion by sako013** [smartcontractkit/full-blockchain-solidity-course-py#1474] [Q&A] [answered]
__pycache__
.env
.history
.hypothesis/
build/
reports/
.vscode/settings.json

this is my .gitigonre content , i wonder if its not okay to push .vscode/ folder to github how can i ignore it cause this is not working for me and .vscode is still in    "git status"

**Comment by sako013** [comment]
this worked 
"git rm --cached .vscode/settings.json "

i wonder if i can delete this post but i could not find it

**Comment by cromewar** [comment, marked as answer]
To add folder you must add a slash "/" so it should be something like:
`.vscode/`  and also yes, is not necessary to push that folder, but some people to to help them on configurations related to the editor, however not everyone uses vscode so is useless for those ones using other editor like sublime.

> i wonder if i can delete this post but i could not find it

Is preferably to leave this post opened in case someone else have the same problem or issue.

Cheers
